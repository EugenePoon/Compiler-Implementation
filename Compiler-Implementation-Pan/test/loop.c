int main()
{
    int i;
    i = 1;
    loop {
        print i;
        if(i == 8)
            break;
        i++;
    }
}