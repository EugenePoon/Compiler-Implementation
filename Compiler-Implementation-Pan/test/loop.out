24 19 0 5 25 15 1 13 0 1 12 15 -1 13 11 22 15 -1 13 11 0 8 6 17 27 16 40 13 11 13 13 11 0 1 1 12 15 -2 16 13 21 0